changelog/v1.20.0-beta6/k8s-133.yaml

```diff
@@ -0,0 +1,62 @@
+changelog:
```

Comment:
nit: let's callout the golang update and potentially link https://github.com/solo-io/solo-projects/issues/8392, since it fixes a vulnerability